### PR TITLE
Document customizations for monitoring charts

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**

Changes proposed in this pull request:

- List all Kyma customizations and compare to upstream for component 'monitoring'

**Related issue(s)**

### CODE REVIEW

File: concepts/observability-automatic-upgrades/customizations.md [hunk 1]
```diff
@@ -10,3 +10,191 @@
...
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#112](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L112) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#127](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L127) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#138](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L138) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#152](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L152) |  |
```
Comment (NHingerl) [inline concepts/observability-automatic-upgrades/customizations.md:57]:
```suggestion
| templates/prometheus/rules-1.14/etcd.yaml | Renamed annotation message to description | [L#152](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L152) |  | 
```

File: concepts/observability-automatic-upgrades/customizations.md [hunk 2]
```diff
@@ -10,3 +10,191 @@
...
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#152](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L152) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#166](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L166) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#180](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L180) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#194](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L194) |  |
```
Comment (NHingerl) [inline concepts/observability-automatic-upgrades/customizations.md:60]:
```suggestion
| templates/prometheus/rules-1.14/etcd.yaml | Renamed annotation message to description | [L#194](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L194) |  | 
```

File: concepts/observability-automatic-upgrades/customizations.md [hunk 3]
```diff
@@ -10,3 +10,191 @@
...
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#180](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L180) |  | 
+| templates/prometheus/rules-1.14/etcd.yaml | renamed annotation message to description | [L#194](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/etcd.yaml#L194) |  | 
+| templates/grafana/dashboards-1.14/alertmanager-overview.yaml | enabled flag | n/a |  | 
+| templates/alertmanager/alertmanager.yaml | images are referenced from global values | [L#15](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/alertmanager/alertmanager.yaml#L15) |  | 
+| templates/grafana/dashboards-1.14/apiserver.yaml | dashboard disabled | n/a |  | 
+| templates/prometheus/prometheus.yaml | images are referenced from global values | [L#34](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L34) |  |
```
Comment (NHingerl) [inline concepts/observability-automatic-upgrades/customizations.md:64]:
```suggestion
| templates/prometheus/prometheus.yaml | Images are referenced from global values | [L#34](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L34) |  | 
```

File: concepts/observability-automatic-upgrades/customizations.md [hunk 4]
```diff
@@ -10,3 +10,191 @@
...
+| templates/grafana/dashboards-1.14/alertmanager-overview.yaml | enabled flag | n/a |  | 
+| templates/alertmanager/alertmanager.yaml | images are referenced from global values | [L#15](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/alertmanager/alertmanager.yaml#L15) |  | 
+| templates/grafana/dashboards-1.14/apiserver.yaml | dashboard disabled | n/a |  | 
+| templates/prometheus/prometheus.yaml | images are referenced from global values | [L#34](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L34) |  | 
+| templates/prometheus/prometheus.yaml | priorityClassName considers global priorityClass | [L#297](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L297) |  | 
+| charts/prometheus-istio/Chart.yaml | deleted home, icon, sources, maintainers, engine, type, dependencies |  |  |
```
Comment (NHingerl) [inline concepts/observability-automatic-upgrades/customizations.md:66]:
```suggestion
| charts/prometheus-istio/Chart.yaml | Deleted home, icon, sources, maintainers, engine, type, dependencies |  |  | 
```

File: concepts/observability-automatic-upgrades/customizations.md [hunk 5]
```diff
@@ -10,3 +10,191 @@
...
+| templates/grafana/dashboards-1.14/alertmanager-overview.yaml | enabled flag | n/a |  | 
+| templates/alertmanager/alertmanager.yaml | images are referenced from global values | [L#15](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/alertmanager/alertmanager.yaml#L15) |  | 
+| templates/grafana/dashboards-1.14/apiserver.yaml | dashboard disabled | n/a |  | 
+| templates/prometheus/prometheus.yaml | images are referenced from global values | [L#34](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L34) |  | 
+| templates/prometheus/prometheus.yaml | priorityClassName considers global priorityClass | [L#297](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L297) |  | 
+| charts/prometheus-istio/Chart.yaml | deleted home, icon, sources, maintainers, engine, type, dependencies |  |  | 
+| charts/prometheus-istio/Chart.yaml | renamed: prometheus-istio |  |  |
```
Comment (NHingerl) [inline concepts/observability-automatic-upgrades/customizations.md:67]:
```suggestion
| charts/prometheus-istio/Chart.yaml | Renamed: prometheus-istio |  |  | 
```

**Comment by pbochynski** [comment]
What it is about? Can you explain the purpose of this document?

**Pull request closed without merging**